src/compressed_tensors/transform/factory/matrix_multiply.py

```diff
@@ -52,14 +52,15 @@ def create_transform(self, module: Module, args: TransformArgs):
         """
         assert hasattr(module, "weight")
         size = get_transform_size(module, args.location, self.scheme.head_dim)
-        dtype = module.weight.dtype
+        dtype = self.scheme.precision
```

Comment:
might as well remove this additional var and just use `self.scheme.precision`?

Reply:
This style is partially to get everything to fit on one line. But I also think this separation is nice because it makes very evident the source of all the information before that information is used.